src/wiringx.c

```diff
@@ -113,6 +113,11 @@ static struct spi_t spi[2] = {
 	} while(0)
 #endif
 
+#ifdef __riscv
```

Comment:
Would the wiringx.h not be a more logical place for this definition?

Reply:
Yes agreed, but with current code we aren't including <time.h> in the header file so it fails to find `time_t`. If we want to move the macros to the header, we have to include time.h there and remove the inclusion from wiringx.c.

https://github.com/wiringX/wiringX/blob/master/src/wiringx.c#L17

Would you like me to do that?

Reply:
If that doesn't have side effects while compiling.

Reply:
Maybe include it within the `__riscv` macro?

Reply:
> If that doesn't have side effects while compiling.

Pushed the changes, works on RISC-V and x86

>  Maybe include it within the __riscv macro?

Can do, though I think since the code I pushed works with both RISC-V and x86, no need to cause any confusion.